src/AltinnCore/Designer/Views/ManualTesting/Index.cshtml

```diff
@@ -58,6 +58,20 @@
       </div>
     </form>
 
+
+    <button id="btnStartNewService" class="btn btn-primary" value="@org/@service">Start ny 2.0</button>
```

Comment:
I guess this button is "temporary". What is "ny 2.0"?

Reply:
It's "Start new" but bumped the version. To 2.0.

Reply:
But should it say.. "Start ny 2.0"? This razor view will still be used by some orgs. The button should state what it actually do.

Reply:
Its basicly the same as the button above, only that this redirects to the service info page on the app. And the other button instantiates an instance on the backend and redirects to that.

Reply:
Orgs will use this page. I think the buttons should state what it does.

Reply:
Do you want the new flow or the old flow on this page then?